agent/bench-scripts/pbench-dbench

```diff
@@ -511,6 +511,7 @@ for thread_count in `echo $thread_counts | sed -e s/,/" "/g`; do
 done
 
 printf -- "</pre>\n" >>$benchmark_summary_html_file
+pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir
```

Comment:
Add "`end`" argument, and invoke it with the "`beg`" argument between lines 303 and 304.